oauth2_provider/templates/oauth2_provider/application_form.html

```diff
@@ -34,7 +34,7 @@ <h3 class="block-center-heading">
                     <a class="btn" href="{% block app-form-back-url %}{% url "oauth2_provider:detail" application.id %}{% endblock app-form-back-url %}">
                         {% trans "Go Back" %}
                     </a>
-                    <button type="submit" class="btn btn-primary">Save</button>
+                    <button type="submit" class="btn btn-primary">{% trans "Save" %}</button>
```

Comment:
Maybe a little more consistency about double/single quotes? I'm not insisting if you feel like your system is better, it's not a big deal either way.